gtkmm fails to build20 aso of 07/02/2020

```
 ./build.sh R       
*** Building Release ***
Making check in po
make[1]: Entering directory '/home/q/prj/cherrytree/future/po'
rm -f missing notexist
srcdir=. /usr/bin/intltool-update -m
The following files contain translations and are currently not in use. Please
consider adding these to the POTFILES.in file, located in the po/ directory.

src/ct/ct_print.cc

If some of these files are left out on purpose then please add them to
POTFILES.skip instead of POTFILES.in. A file 'missing' containing this list
of left out files has been written in the current directory.
Please report to giuspen@gmail.com
if [ -r missing -o -r notexist ]; then \
  exit 1; \
fi
Makefile:173: recipe for target 'check' failed
make[1]: *** [check] Error 1
make[1]: Leaving directory '/home/q/prj/cherrytree/future/po'
Makefile:2185: recipe for target 'check-recursive' failed
make: *** [check-recursive] Error 1

```

--- a/future/po/POTFILES.in
+++ b/future/po/POTFILES.in
@@ -23,4 +23,7 @@ src/ct/ct_actions_view.cc
 src/ct/ct_actions_format.cc
 src/ct/ct_actions_edit.cc
 src/ct/ct_actions_others.cc
+src/ct/ct_actions_export.cc
+src/ct/ct_print.cc
+src/ct/ct_export2html.cc
 src/fmt/format.h

